pkg/operator/ceph/webhook-config.go

```diff
@@ -166,11 +169,15 @@ func fetchValidatingWebhookConfig(ctx context.Context, clusterdContext *clusterd
 }
 
 func createValidatingWebhookConfig(ctx context.Context, clusterdContext *clusterd.Context) error {
-	sideEffects := admv1.SideEffectClassNone
-	var timeout int32 = 5
-	serviceCephClusterPath := "/validate-ceph-rook-io-v1-cephcluster"
-	serviceCephBlockPoolPath := "/validate-ceph-rook-io-v1-cephblockpool"
-	serviceCephObjectStorePath := "/validate-ceph-rook-io-v1-cephobjectstore"
+	name := []string{
```

Comment:
Give the variable a better name?

Reply:
modified, naming is always a challenge :slightly_smiling_face: